packages/core/src/Menu/MenuSub.vue

```
@@ -1,6 +1,8 @@
 <script lang="ts">
 import type { Ref } from 'vue'
 import type { MenuContext } from './MenuRoot.vue'
+import type { MenuOpenChangeReason } from './useMenu'
+import type { ChangeEventDetails } from '@/shared'
 import { createContext } from '@/shared'
```

ESLint

[error] '`@/shared`' imported multiple times.

(import/no-duplicates)